### ISSUE TITLE
Hide apps based on focus mode

### ISSUE THREAD

**Issue by jbytes1027** [8bitPit/Niagara-Issues#1464]
**Describe the solution you'd like**
hide apps that are blocked by current digital wellbeing focus mode

**Comment by 8bitPit** [comment]
I think that's a duplicate of: https://github.com/8bitPit/Niagara-Issues/issues/1119
